source/renderer/app/components/loading/Loading.js

```diff
@@ -149,6 +151,7 @@ type Props = {
   disableDownloadLogs: boolean,
 };
 
+@inject('actions')
```

Comment:
Components cannot access the `actions` and `stores` directly. Please, add `openDaedalusDiagnosticsDialog` as a Prop at `Loading` and pass it down from `LoadingPage`.